src/Repository/UserRepository.php

```diff
@@ -8,15 +8,19 @@
 use Symfony\Component\Security\Core\Exception\UnsupportedUserException;
 use Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface;
 use Symfony\Component\Security\Core\User\PasswordUpgraderInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
 
 /**
  * @extends ServiceEntityRepository<User>
  */
 class UserRepository extends ServiceEntityRepository implements PasswordUpgraderInterface
 {
-    public function __construct(ManagerRegistry $registry)
+    private TranslatorInterface $translator;
+
+    public function __construct(ManagerRegistry $registry, TranslatorInterface $translator)
```

Comment:
This will give an error, please fix this

```suggestion
    public function __construct(ManagerRegistry $registry, private TranslatorInterface $translator)
```

Reply:
resolved by @Jonatas-Soares-Alves